src/handlers/handleCLIDownloader.ts

```diff
@@ -252,6 +313,37 @@ class DependenciesDownloader {
             logger.debug("CBShell is already installed");
             this.setToolActive(ToolStatus.AVAILABLE, shellPath, shell);
         }
+
+        const cbMigrate = downloads.get(this.TOOL_MDB_MIGRATE);
+        if (cbMigrate === undefined) {
+            return;
+        }
+        const cbMigratePath = path.join(
+            toolsPath,
+            cbMigrate.getInstallationPath()
+        );
+        const cbMigrateTool = CBTools.getTool(CBToolsType.CB_MIGRATE);
+        const cbMigrateStatus = cbMigrateTool.status;
+        const cbMigrateDownloadsMap = downloads.get(this.TOOL_SHELL);
+        if (cbMigrateDownloadsMap === undefined) {
+            return;
+        }
+        if (
+            cbMigrateStatus === ToolStatus.NOT_AVAILABLE &&
+            !this.isInstalled(
+                toolsPath,
+                cbMigrateDownloadsMap,
+                CBToolsType.CB_MIGRATE
+            )
+        ) {
+            // Avoiding 2 threads to install the same thing at the same time
+            logger.info("Downloading CB Migrate..");
```

Comment:
Use single full stop at the end. Or add multiple to other tools as well

Reply:
Updated.